src/openrct2/peep/Staff.cpp

```diff
@@ -1173,8 +1169,8 @@ void Staff::UpdateMowing()
             return;
         }
 
-        DestinationX = _MowingWaypoints[Var37].x + NextLoc.x;
-        DestinationY = _MowingWaypoints[Var37].y + NextLoc.y;
+        auto destination = _MowingWaypoints[Var37] + NextLoc;
+        SetDestination(destination);
```

Comment:
Assume this will be tolerance 3 as that's what it was too get to the first waypoint. Maybe do a follow on pr that forces setting the tolerance.

Reply:
I initially did that where default was zero however that broke the replays as it changes the hash so for testing this was better.